docs/python/reference/dbos-class.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_position: 1
+title: DBOS Class
+---
+
+The DBOS class is a singleton&mdash;you may only instantiate it once in a program's lifetime.
+Here, we document its constructor and lifecycle methods.
+Decorators are documented [here](./decorators.md) and context methods and variables [here](./contexts.md).
+
+### class dbos.DBOS
+
+```python
+DBOS(
+    *,
+    fastapi: Optional[FastAPI] = None,
+    config: Optional[ConfigFile] = None,
+)
+```
+
+**Parameters:**
+- `fastapi`: If your application is using FastAPI, the `FastAPI` object. If this is passed in, DBOS automatically calls `launch` when FastAPI is fully initialized. DBOS also adds to all routes a middleware that enables [tracing](../tutorials/logging-and-tracing.md#tracing) through FastAPI HTTP endpoints.
+- `config`: A configuration object, which if passed in is used in place of `dbos-config.yaml`. We recommend using this for testing only.
+
+
+### launch
+
+```python
+dbos.launch()
+```
+
+Launch DBOS, initializing database connections and starting scheduled workflows.
+Should be called in your main function.
+If a FastAPI app is passed into the `DBOS` constructor, `launch` is called automatically during FastAPI setup.
+
+**Example:**
+```python
+from dbos import DBOS
+
+# Initialize the DBOS object
+dbos = DBOS()
+
+# Define a scheduled workflow
+@DBOS.scheduled("* * * * *")
+@DBOS.workflow()
+def run_every_minute(scheduled_time: datetime, actual_time: datetime):
+    DBOS.logger.info("This is a scheduled workflow!")
+
+# In your main function, launch DBOS
+if __name__ == "__main__":
+    dbos.launch()
```

Comment:
It does look weird that we use `dbos.launch()` but `DBOS.destroy()`. This small dbos vs big DBOS still look weird. Why can't we just use `DBOS.launch()`?